doc/contents.rst

```diff
@@ -8,6 +8,7 @@ Salt Table of Contents
     :maxdepth: 2
 
     topics/index
+	topic/community
```

Comment:
I don't think this is _quite_ right. I just built the docs with the changes in this PR. While I can access the new community page via the direct link, I don't see it on the table of contents.

There is also a related warning generated when building the docs:
```
/root/SaltStack/salt/doc/contents.rst:7: WARNING: toctree contains reference to nonexisting document u'    topic/community'
```

Reply:
@barbaricyawps You just need to align this with `topics/index`. Also, this should be `topics/community/index` and not `topic/community`.